addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2965,6 +2967,9 @@ export class OdooEditor extends EventTarget {
                 cleanZWS(emptyElement);
                 emptyElement.removeAttribute('oe-zws-empty-inline');
             }
+            if(blockEl && isShrunkBlock(blockEl)) {
+                fillEmpty(blockEl);
+            }
```

Comment:
Don't `fillEmpty` check if block is `isShrunkBlock`?
Also there is a linting issue with if block.